platalea/utils/preprocessing.py

```diff
@@ -17,33 +17,33 @@
 import torchvision.transforms as transforms
 from platalea.experiments.config import get_argument_parser
 
+# Number of files to keep in the debug mode
+NB_DEBUG = 50
 
-args = get_argument_parser()
-_audio_feat_config = dict(type='mfcc', delta=True, alpha=0.97, n_filters=40,
-                          window_size=0.025, frame_shift=0.010)
-_images_feat_config = dict(model='resnet')
 
-
-def preprocess_flickr8k(dataset_path, audio_subdir, image_subdir):
+def preprocess_flickr8k(dataset_path, audio_subdir, image_subdir,
+                        _audio_feat_config, _images_feat_config):
```

Comment:
Could you rename the last two parameters, removing the starting underscore? Same goes for the rest of the file.